_posts/2023-05-25-10block-old-decoy-selection-bug.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: Post-Mortem of 10-Block-Old Decoy Selection Bug
+summary: Patched in official wallet, update highly recommended.
```

Comment:
Nit: I'd call it the "reference wallet" instead of "official" since there are other "official" wallets :)